src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -260,28 +231,21 @@ async def count_dispatcher(self) -> AsyncIterator[None]:
     async def handle_dispatch(self, websocket: WebSocketServerProtocol) -> None:
         async with self.count_dispatcher():
             try:
-                async for msg in websocket:
-                    try:
-                        event = from_json(msg, data_unmarshaller=evaluator_unmarshaller)
-                    except cloudevents.exceptions.DataUnmarshallerError:
-                        event = from_json(msg, data_unmarshaller=pickle.loads)
-                    if self._get_ens_id(event["source"]) != self.ensemble.id_:
-                        logger.info(
-                            "Got event from evaluator "
-                            f"{self._get_ens_id(event['source'])} "
-                            f"with source {event['source']}, "
-                            f"ignoring since I am {self.ensemble.id_}"
-                        )
-                        continue
+                async for raw_msg in websocket:
                     try:
-                        if event["type"] == EVTYPE_FORWARD_MODEL_CHECKSUM:
+                        event = cast(DispatchEvent, event_from_json(raw_msg))
```

Comment:
Can you use directly pydantic func like `Dispatchevent.parse_raw(raw_msg)`?

Reply:
made a new dispatch_event_from_json. do you like that better?

Reply:
https://github.com/equinor/ert/blob/3acdf621f2c8bbf1ddabb43087595885f3012ea5/src/_ert/events.py#L221-L229